NightscoutServiceKit/NightscoutService.swift

```diff
@@ -122,6 +130,14 @@ public final class NightscoutService: Service {
         try? KeychainManager().setNightscoutCredentials()
     }
     
```

Comment:
For the override issue mentioned above.

Limitation - this wouldn't handle the case of changing the site url or apiSecret.